apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -646,6 +666,12 @@ const refreshProject = async () => {
   toast.error(result.error);
 };
 
+const buttonLinkClass = css({
```

Comment:
need to create a button link component to avoid this mess, but for now lets stick with how its done everywhere else:

```
          <Link
            className={buttonStyle({color: "neutral"})}
            underline="none"
            href={}
            target="_blank"
          >
            Open project
          </Link>
```

Reply:
that's weird, you could have just used a link and made prevent default :)

Reply:
There is no url to navigate to